src/gretel_trainer/relational/multi_table.py

```diff
@@ -722,15 +726,39 @@ def run_transforms(
         self._backup()
         self.transform_output_tables = reshaped_tables
 
+    def _get_only_and_ignore(
+        self, only: Optional[list[str]], ignore: Optional[list[str]]
+    ) -> tuple[Optional[list[str]], Optional[list[str]]]:
```

Comment:
nit: these table filters could be sets, right? As the order doesn't matter?

Reply:
Yes for sure, and passing a particular table name multiple times makes no sense. I actually used lists figuring most users would find lists slightly more intuitive/familiar than sets, but hinting with set is probably more informative to anyone looking at the code / documentation, + if you pass a list anyways it'll still work since all we're doing is iterating over the collection. Pushing a commit for this.